src/components/MultiLink/Legend.vue

```diff
@@ -123,105 +126,98 @@ export default Vue.extend({
           // eslint-disable-next-line @typescript-eslint/no-explicit-any
           let currentData: any[];
 
-          if (type === 'node') {
-            currentData = this.graphStructure.nodes.map((d: Node | Link) => d[attr]);
+          let xScale: ScaleLinear<number, number> | ScaleBand<string> | null = null;
+          let yScale: ScaleLinear<number, number> | null = null;
+          let bins: Bin<number, number>[] | Map<string, number> | null = null;
+
+          // Process data for bars/histogram
+          if (this.isQuantitative(attr, type)) {
+            if (type === 'node') {
+              currentData = this.graphStructure.nodes.map((d: Node | Link) => parseFloat(d[attr]));
+            } else {
+              currentData = this.graphStructure.edges.map((d: Node | Link) => parseFloat(d[attr]));
+            }
+
+            // TODO: add histogram code
```

Comment:
Isn't that what this PR does?

Reply:
Yes, this was a remnant from cleaning up the commit stream. I left it in accidentally. It's removed in 8876d59